CHANGES

```diff
@@ -1,18 +1,13 @@
 # Changelog
 
-Version 0.2.0 (Public Preview)
-
+## Version 0.2.0 (Public Preview) - 2021-08-30
+### Added
   - Potential Breaking Change: Loading of XML external entities are not loaded by default. Users must explictly allow it through using the new "allowXmlUnsafeExternalEntity" connection URL parameter. It is recommended that users verify external entities before loading them.
-
   - Added "setAwsProtocolOnly" static method.
-
   - Merged upstream changes from MySQL 8.0.23 community driver.
-
   - Added clarifications and improvements to README.md.
 
-Version 0.1.0 (Public Preview)
-
+## Version 0.1.0 (Public Preview) - 2021-01-06
```

Comment:
https://github.com/awslabs/amazon-qldb-driver-java/blob/master/CHANGELOG.md
Example from QLDB, we can follow their example for version 2.0.0.
Don't forget to link the tags.